src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -778,6 +802,7 @@ class HttpGoogleServicesDAO(
   def toGoogleGroupName(groupName: RawlsGroupName) = s"GROUP_${groupName.value}@${appsDomain}"
 
   def adminGroupName = s"${groupsPrefix}-ADMINS@${appsDomain}"
+  def curatorGroupName = s"${groupsPrefix}-CURATORS@${appsDomain}"
```

Comment:
I created it manually in all envs except for prod. I don't think this group necessarily deserves different treatment than groups like fc-ADMINS and dbGapAuthorizedUsers but if we want to start a trend of doing these things automatically I'm fine with that